docs/java-rest/high-level/security/invalidate-token.asciidoc

```diff
@@ -1,39 +0,0 @@
---
```

Comment:
I assume this is being removed because the snippets aren't running.

Normally, I think it would be better to remove this doc from the include file, or remove the snippet-inclusion rather than delete it.